pkgs/by-name/ma/magicard-cups-driver/package.nix

```
@@ -0,0 +1,171 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  cmake,
+  cups,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "magicard-cups-driver";
+  version = "1.4.0";
+
+  src = fetchurl {
+    url = "https://f08ddbe93aa02eaf9a6c-f08cd513e3a8c914f4f8f62af1786149.ssl.cf3.rackcdn.com/magicard_ltd-linux_driver-${version}.tar.gz";
+    hash = "sha256-OAdpxnGMqVcf3bEtYQ9PACZFJvq8gzhGxuBBnliwoso=";
```

with changing the fetcher the hash also changes

```suggestion
    hash = "sha256-1k2Twn1JBizw/tzQ0xF1uJIecblRd6VurB7FAUop5F0=;
```